### ISSUE TITLE
"Make flowchart" does not escape special characters in job title

### ISSUE THREAD

**Issue by rdrighetto** [3dem/relion#444]
When clicking on a finished job on the GUI and selecting the option "Make flowchart", the flowchart is not generated if the job alias contains special characters such as "_".
I noticed that the latex command was taking forever to finish. When trying to run it manually on the `flowchart.tex` file, it would get stuck in the interactive error message:

```
! Missing $ inserted.
<inserted text>
                $
l.10 ...R_postprocess_3dref12b_Pmax0.1_soft6_BEST}
```
As you can see, my job alias contains underscores. After manually editing `flowchart.tex` to escape these characters in line 10 (replace "\_" with "\\_"), I was able to successfully generate the flowchart.
So, it would be nice if the RELION pipeliner would automatically take care of this :-)

Thank you!

**Comment by biochem-fan** [comment]
This makes sense. Thanks for pointing it out.

Previously I have added checks for special characters that might cause problems with the shell (like `<`) but never thought about LaTeX.

**Comment by biochem-fan** [comment]
Fixed in 06b59a7.
